.gitignore

```diff
@@ -89,12 +89,13 @@ dist
 .vscode-test
 
 # CNCF Elements
-Kubestronaut.tsv
-Coupons.csv
-Ambassadors.tsv
-*-service-file.json
 python_venv/
 python_env_cncfpeople
 python_venv_ambassadors
-credentials.json
-token.json
+**/*.tsv
+**/*.csv
+**/*.svg
+**/*.png
+**/*.json
+**/token
```

Comment:
@huats I was looking at this list of file types working thru it to consider if they had any impact on the external GHA work and I was thinking that it would probably be fine. 

But then I noticed that there is a json file in this PR (suggestions.json) are you sure you want to have json files .gitgnore moving forward?